web/env.html

```diff
@@ -110,6 +112,7 @@
                 <a id="helpButton" class="cw-button blue" target="_blank" href="http://help.code.world"><i class="mdi mdi-18px mdi-comment-question-outline"></i>&nbsp;
                     Ask a Question</a>
                 <a id="bugButton" class="cw-button blue" target="_blank" href="https://github.com/google/codeworld/issues/new?template=bug_report.md"><i class="mdi mdi-18px mdi-bug"></i>&nbsp; Report a Bug</a>
+                <a id="toggleThemeButto" class="cw-button blue theme-marker-day" title="Toggle theme" target="_blank" onclick="toggleTheme()"><i class="mdi mdi-18px mdi-weather-sunny"></i></a>
```

Comment:
Misspelling in the `id` atttribute.  It doesn't look like you even need the id anyway.

Reply:
True. Deleting this.